meson_options.txt

```diff
@@ -34,6 +34,8 @@ option('debug-shell', type : 'string', value : '/bin/sh',
        description : 'path to debug shell binary')
 option('debug-tty', type : 'string', value : '/dev/tty9',
        description : 'specify the tty device for debug shell')
+option('extra-debug', type : 'string',
+       description : 'enable extra debugging (hashmap,mmap-cache)')
```

Comment:
On second thought, I decided to rename the meson option. `-Ddebug=hashmap` is verbose enough. No it matches ./configure ;)